server.ts

```diff
@@ -50,9 +51,21 @@ export function app(path: string): express.Express {
     })
   );
 
+  // Gzip static assets
+  server.use(compressionModule());
```

Comment:
We use nginx for this in production. 

We should be ensuring we're using x-sendfile though to serve assets (https://www.nginx.com/resources/wiki/start/topics/examples/xsendfile/) and follow up in our deploy playbooks. For another time though.